src/components/App/App.tsx

```diff
@@ -1,98 +1,83 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import Feed from '../Feed/Feed';
 import Form from '../Form/Form';
 import {BrowserRouter as Router, Route, Switch} from 'react-router-dom';
 import './App.css';
 import { CleanArticle, fetchNewsData, getSentiment } from '../../apiCalls'
 
-interface AppState {
-  articles: CleanArticle[],
-  userSentiment: number | string
-}
 
-interface AppProps {}
+const App = (): JSX.Element => {
 
-class App extends React.Component<AppProps, AppState> {
-  constructor(props: AppProps) {
-    super(props)
-    this.state = {
-      articles: [],
-      userSentiment: ''
-    }
-  }
+  const [ articles, setArticles ] = useState<CleanArticle[]>([])
+  const [ userSentiment, setUserSentiment ] = useState<number | null>(null)
```

Comment:
What do you think about omitting  userSentiment from state? We aren't changing anything on the DOM based off of the userSentiment changing in our application right now and this would get rid of the error we are getting that we are "userSentiment' is defined but never used"? You'd basically delete line 11 and then remove the invocation of setUserSentiment in changeUserSentiment. Since these user sentiment numbers are being injected  as arguments into the assignSentiment function in Form, everything should still function appropriately...I believe!

Reply:
An alternative would be to have `changeUserSentiment` do only one thing: call `setUserSentiment` with `newSentiment`. Then, we can reach for `useEffect` to watch `userSentiment` and call `sortBySentiment` whenever it changes. Something like this:

```
  useEffect((): void => {
    sortBySentiment(userSentiment)
  }, [userSentiment])

  const changeUserSentiment = (newSentiment: number) => {
    setUserSentiment(newSentiment)
  }
```

Reply:
I went with the useEffect approach because at some point we talked about trying to track useSentiment and its changes so maybe it will be good to have it stored in our App component.